appveyor.ps1

```diff
@@ -12,7 +12,7 @@ $env:compilersetup = "C:\Program Files (x86)\Microsoft Visual Studio 14.0\VC\vcv
 
 if ($env:arch -eq "x86") {
   $env:compilersetupargs = "x86";
-  $env:archswitch = "";
+  $env:archswitch = "--arch=x86_mscoff";

```

Comment:
In line 45 of the same file. I'm not sure if it is *strictly* necessary at the moment (maybe test.d can be compiled without libclang?) but I think it should be present just not to mix `x86` and `x86_mscoff` architectures.